test/integration/helpers.go

```diff
@@ -111,6 +112,40 @@ func Run(t *testing.T, cmd *exec.Cmd) (*RunResult, error) {
 	return rr, err
 }
 
+// RunInPowershell is a test helper to log a command being executed in powershell \_(ツ)_/¯
```

Comment:
Please replace this helper with a properly specified *cmd.Cmd structure. It's doing so much opaque and unnecessary work from a testing perspective.

Reply:
done